codespell_lib/data/dictionary.txt

```diff
@@ -9595,6 +9595,16 @@ converterd->converted, converter,
 convertet->converted
 convertion->conversion
 convertions->conversions
+convervable->conservable
+convervation->conservation, conversation,
+convervative->conservative
+convervatives->conservatives
+converve->conserve, converse,
+converved->conserved, conversed,
+converver->conserver
```

Comment:
Maybe `converter` too? But then such a typo is much less likely.

Reply:
OK, we can do that. Now done.